src/test/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStepTest.java

```diff
@@ -2,12 +2,10 @@
 
 import org.jenkinsci.plugins.workflow.steps.Step;
 import org.jenkinsci.plugins.workflow.steps.StepConfigTester;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.jvnet.hudson.test.JenkinsRule;
 
-@Ignore
```

Comment:
Yaa! it was a dependency issue...

Reply:
Well tests are now actually failing due to null 😭 Again on Linux only, WTF. They work on my Mac and Windows.